webapp/package.json

```diff
@@ -83,7 +83,7 @@
     "v-tooltip": "~2.0.2",
     "validator": "^12.0.0",
     "vue-count-to": "~1.0.13",
-    "vue-infinite-scroll": "^2.0.2",
+    "vue-infinite-loading": "^2.4.4",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-15T13:37:51Z" title="Friday, November 15th 2019, 2:37:51 pm +01:00">Nov 15, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -83,6 +83,7 @@
     "v-tooltip": "~2.0.2",
     "validator": "^12.0.0",
     "vue-count-to": "~1.0.13",
+    "vue-infinite-loading": "^2.4.4",
     "vue-infinite-scroll": "^2.0.2",
```


```suggestion
```

Run `yarn remove vue-infinite-loading`

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-15T14:19:37Z" title="Friday, November 15th 2019, 3:19:37 pm +01:00">Nov 15, 2019</time>_

---


I don't want to remove `vue-infinite-loading`, I just added it cause it is being maintained, whereas `vue-infinite-scroll` is not.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-15T14:19:47Z" title="Friday, November 15th 2019, 3:19:47 pm +01:00">Nov 15, 2019</time>_

---


I think I got the idea though... hehe